cities/crud.py

```diff
@@ -0,0 +1,82 @@
+from typing import List
+
+from fastapi import HTTPException, status
+
+from sqlalchemy.orm import Session
+
+from dependencies import ListDep, CityItemDep
+from models import DbCity
+from schemas import CityBase, CityCreate, City
```

Comment:
The import statement for schemas should specify the correct module path. Ensure that `CityBase`, `CityCreate`, and `City` are imported from the correct module, possibly `cities.schemas`.